app/src/main/java/com/kickstarter/ui/viewholders/UnansweredSurveyViewHolder.java

```diff
@@ -0,0 +1,92 @@
+package com.kickstarter.ui.viewholders;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.Html;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.kickstarter.R;
+import com.kickstarter.libs.KSString;
+import com.kickstarter.libs.transformations.CircleTransformation;
+import com.kickstarter.libs.utils.ObjectUtils;
+import com.kickstarter.models.SurveyResponse;
+import com.kickstarter.viewmodels.UnansweredSurveyHolderViewModel;
+import com.squareup.picasso.Picasso;
+
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.BindString;
+import butterknife.ButterKnife;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
+
+public class UnansweredSurveyViewHolder extends KSViewHolder {
+
+  private final KSString ksString;
+  @Bind(R.id.survey_avatar_image) ImageView creatorAvatarImageView;
+  @Bind(R.id.survey_title) TextView surveyTitleTextView;
+  @Bind(R.id.survey_text) TextView surveyTextView;
+
+  @BindString(R.string.Creator_name_needs_some_information_to_deliver_your_reward_for_project_name) String surveyDescriptionString;
+
+  private final UnansweredSurveyHolderViewModel.ViewModel viewModel;
+
+  private final Delegate delegate;
```

Comment:
can move all these `private final` variables to top of class for some organization!